### PR TITLE
Display a message if no Group is found

### PR DESCRIPTION
**Pull request by arjitc**
Screenshot: http://i.imgur.com/ac9okNO.png

### CODE REVIEW

File: html/pages/device-groups.inc.php [hunk 1]
```diff
@@ -3,23 +3,31 @@
...
+	    echo '<td>'.$group['name'].'</td>';
+	    echo '<td>'.$group['desc'].'</td>';
+	    echo '<td>'.$group['pattern'].'</td>';
+	    echo '<td>';
+	        echo "<button type='button' class='btn btn-primary btn-sm' aria-label='Edit' data-toggle='modal' data-target='#create-group' data-group_id='".$group['id']."' name='edit-device-group'><span class='glyphicon glyphicon-pencil' aria-hidden='true'></span></button> ";
+	        echo "<button type='button' class='btn btn-danger btn-sm' aria-label='Delete' data-toggle='modal' data-target='#confirm-delete' data-group_id='".$group['id']."' name='delete-device-group'><span class='glyphicon glyphicon-trash' aria-hidden='true'></span></button>";
+	    echo '</td>';
+	    echo '</tr>';
+	}
+} else { //if $group_count_check is empty, aka no group found, then display a message to the user.
```
Comment (laf) [inline html/pages/device-groups.inc.php]: Can you update this to:

}
else {

File: html/pages/device-groups.inc.php [hunk 2]
```diff
@@ -3,23 +3,31 @@
...
+	echo "<center>Looks like no groups have been created, let's create one now. Click on <b>Create New Group</b> to create one.</center><br>";
+	echo "<center><button type='button' class='btn btn-primary btn-sm' aria-label='Add' data-toggle='modal' data-target='#create-group' data-group_id='' name='create-device-group'>Create new Group</button></center>";
+}	
 echo '</tbody></table></div>';
-echo "<button type='button' class='btn btn-primary btn-sm' aria-label='Add' data-toggle='modal' data-target='#create-group' data-group_id='' name='create-device-group'>Create new Group</button> ";
+
+if(!empty($group_count_check)) { //display create new node group when $group_count_check has a value so that the user can define more groups in the future.
+echo "<hr>";
```
Comment (laf) [inline html/pages/device-groups.inc.php]: Need to indent line 31 + 32 with 4 spaces please.

**Comment by laf** [comment]
@arjitc looks good to me apart from those couple of coding style updates to be standard.

**Comment by arjitc** [comment]
@laf fixed :)

**Pull request merged**